src/openloco/localisation/string_ids.h

```diff
@@ -75,6 +116,82 @@ namespace openloco::string_ids
+    constexpr string_id gettin_on_the_gas_credit = 1027;
+    constexpr string_id jumpin_the_rails = 1028;
+    constexpr string_id jumpin_the_rails_credit = 1029;
+    constexpr string_id smooth_running = 1030;
+    constexpr string_id smooth_running_credit = 1031;
+    constexpr string_id traffic_jam = 1032;
+    constexpr string_id traffic_jam_credit = 1033;
+    constexpr string_id never_stop_til_you_get_there = 1034;
+    constexpr string_id never_stop_til_you_get_there_credit = 1035;
+    constexpr string_id soaring_away = 1036;
+    constexpr string_id soaring_away_credit = 1037;
+    constexpr string_id techno_torture = 1038;
+    constexpr string_id techno_torture_credit = 1039;
+    constexpr string_id everlasting_high_rise = 1040;
+    constexpr string_id everlasting_high_rise_credit = 1041;
+    constexpr string_id solace = 1042;
+    constexpr string_id solace_credit = 1043;
+    constexpr string_id chrysanthemum = 1044;
+    constexpr string_id chrysanthemum_credit = 1045;
+    constexpr string_id eugenia = 1046;
+    constexpr string_id eugenia_credit = 1047;
+    constexpr string_id the_ragtime_dance = 1048;
+    constexpr string_id the_ragtime_dance_credit = 1049;
+    constexpr string_id easy_winners = 1050;
+    constexpr string_id easy_winners_credit = 1051;
+    constexpr string_id setting_off = 1052;
+    constexpr string_id setting_off_credit = 1053;
+    constexpr string_id a_travellers_seranade = 1054;
+    constexpr string_id a_travellers_seranade_credit = 1055;
+    constexpr string_id latino_trip = 1056;
+    constexpr string_id latino_trip_credit = 1057;
+    constexpr string_id a_good_head_of_steam = 1058;
+    constexpr string_id a_good_head_of_steam_credit = 1059;
+    constexpr string_id hop_to_the_bop = 1060;
+    constexpr string_id hop_to_the_bop_credit = 1061;
+    constexpr string_id the_city_lights = 1062;
+    constexpr string_id the_city_lights_credit = 1063;
+    constexpr string_id steamin_down_town = 1064;
+    constexpr string_id steamin_down_town_credit = 1065;
+    constexpr string_id bright_expectations = 1066;
+    constexpr string_id bright_expectations_credit = 1067;
+    constexpr string_id mo_station = 1068;
+    constexpr string_id mo_station_credit = 1069;
+    constexpr string_id far_out = 1070;
+    constexpr string_id far_out_credit = 1071;
+    constexpr string_id running_on_time = 1072;
+    constexpr string_id running_on_time_credit = 1073;
+    constexpr string_id get_me_to_gladstone_bay = 1074;
+    constexpr string_id get_me_to_gladstone_bay_credit = 1075;
+    constexpr string_id chuggin_along = 1076;
+    constexpr string_id chuggin_along_credit = 1077;
+    constexpr string_id dont_lose_your_rag = 1078;
+    constexpr string_id dont_lose_your_rag_credit = 1079;
+    constexpr string_id sandy_track_blues = 1080;
+    constexpr string_id sandy_track_blues_credit = 1081;
+
+    // Unused
+    constexpr string_id atari_inc_credits_btn = 1929;
```

Comment:
Move the `// Unused` to after the line. E.g. `…its_btn = 1929; // Unused`

Reply:
Surely `clang-format` won't like that?

Reply:
Better idea: 
```
#define UNUSED_STR(x) (x)
constexpr string_id atari_inc_credits_btn = UNUSED_STR(1929);
```

That way clang format keeps it clean, and it's easily findable